components/list/README.md

```diff
@@ -78,6 +78,41 @@ The `d2l-list-item` provides the appropriate `listitem` semantics for children w
 - `selectable` (Boolean): Indicates a checkbox should be rendered for selecting the item
 - `selected` (Boolean): Whether the item is selected
 
+## ListItemMixin
+
+What if you want to use a list item need more modularity? This mixin is for you! This mixin allows you to make a component a list item without the worry of styling. All the properties from `d2l-list-item` will be added to your new component.
+
+### How to use
+
+Import
+```javascript
+import { ListItemMixin } from './list-item-mixin.js';
+
+class ListItem extends ListItemMixin(LitElement) {
+...
+```
+
+How add the styles:
+```javascript
+static get styles() {
+	return [ super.styles ];
```

Comment:
I think this is only necessary if people need to extend the css on the component.
```suggestion
	return [ super.styles, css`
	    .some-custom-class {
	        padding: 1rem;
	     }
	` ];
```

Reply:
It's true. However, I sort of want to show how people can make their own. So that is why I added it.